lib/internal/types.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+const ReflectApply = Reflect.apply;
+
+function uncurryThis(func) {
+  return (thisArg, ...args) => ReflectApply(func, thisArg, args);
+}
+
+function getterFunc(obj, prop) {
+  return Object.getOwnPropertyDescriptor(obj, prop).get;
+}
+
+const TypedArrayPrototype = Object.getPrototypeOf(Uint8Array.prototype);
+
+const TypedArrayProto_toStringTag =
+    uncurryThis(getterFunc(TypedArrayPrototype, Symbol.toStringTag));
+
+const isArrayBufferView = ArrayBuffer.isView;
```

Comment:
Would you be so kind and add a comment that you added this to make sure it will not is not manipulated by userland code?